components/Player.js

```diff
@@ -3,23 +3,44 @@ import Show from './Show';
 import formatTime from '../lib/formatTime';
 
 export default class Player extends React.Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
+
+    var lastPlayed = 0;
+
+    if (typeof window !== 'undefined') {
+      const lp = localStorage.getItem('lastPlayed' + this.props.show.number);
+      if(lp) lastPlayed = JSON.parse(lp).lastPlayed;
+    }
+
     this.state = {
       progressTime: 50,
       playing: false,
       duration: 0,
-      currentTime: 0,
-      playbackRate: 1
+      currentTime: lastPlayed,
+      playbackRate: 1,
+      timeWasLoaded: lastPlayed !== 0,
     }
+
   }
 
   timeUpdate = (e) => {
+    // Check if the user already had a curent time
+    if(this.state.timeWasLoaded) {
+      const lp = localStorage.getItem('lastPlayed' + this.props.show.number);
+      if(lp) {
+        currentTime = JSON.parse(lp).lastPlayed;
+      }
+      this.state.timeWasLoaded = false;
```

Comment:
This is mutating state directly - use this.setState({ timeWasLoaded : false }) instead